packages/sui-theme/src/settings/_spacing.scss

```diff
@@ -30,6 +30,7 @@ $m-l: $m-base * 2 !default; // 16px
 $m-m: $m-base !default; // 8px
 $m-s: $m-base / 2 !default; // 4px
 $m-xs: $m-base / 4 !default; // 2px
+$m-xxs: 1px !default;
```

Comment:
IMO, all big sizes are relative to $m-base and it might also be as them.
`$m-xxs: $m-base / 8 !default; // 1px`

it might be also spreaded to paddings and add:
$p-xxs: $p-base / 8 !default; // 1px